.github/workflows/android.yml

```
@@ -0,0 +1,70 @@
+
+      - name: StelPlay Android CI Discord Notification
+        uses: sarisia/actions-status-discord@v1
+        if: ${{ failure() }}
+        with:
+          title: ❗️ StelPlay-Android-CI 실패! ❗️
+          webhook: ${{ secrets.STELPLAY_DISCORD_WEBHOOK }}
+          status: ${{ job.status }}
+          image: ${{ secrets.FAILED_IMAGE }}
+          color: 00FF00
+          url: "https://github.com/sarisia/actions-status-discord"
+          username: StelPlay CI Bot
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)